web/static/js/app.js

```diff
@@ -34,64 +34,72 @@ var gameWorld;
+  player.anchor.setTo(.5, .5);
+  game.physics.enable(player, Phaser.Physics.ARCADE);
+  player.body.setCircle(player.width / 2, 0, 0);
+  player.body.debug = true;
 
-    game.world.setBounds(0, 0, 2500, 2500);
-    game.camera.follow(player);
-    game.camera.bounds = null;
-    // gameWorld.scale.set(.5);
+  game.world.setBounds(0, 0, 2500, 2500);
+  game.camera.follow(player);
+  game.camera.bounds = null;
+  // gameWorld.scale.set(.5);
 
+  gameChannel.on("state_tick", ({ state }) => {
+    console.log(JSON.stringify(state));
+  })
 }
 
 const trailInterval = 5;
 var trailCounter = trailInterval;
 
 function update() {
 
-    // Move/turn toward pointer
-    game.physics.arcade.moveToPointer(player, 400);
-    player.rotation = game.physics.arcade.angleToPointer(player);
-
-    // Stop on mouseover
-    if (Phaser.Rectangle.contains(player.body, game.input.worldX, game.input.worldY))
-    {
-        player.body.velocity.setTo(0, 0);
-    }
-
-    if (trailCounter++ > trailInterval) {
-        trailCounter = 0;
-
-        // Draw trail
-        var trail = game.add.graphics(0, 0);        
-        for (let x of [0-game.world.bounds.width, 0, game.world.bounds.width])
-            for (let y of [0-game.world.bounds.height, 0, game.world.bounds.height]) {
-                trail.beginFill(0xffffff);
-                trail.drawCircle(x + player.x, y + player.y, 3);
-                trail.endFill();
-            }
-        setTimeout(() => trail.kill(), 3000); // Keep trail 3 seconds long
-    }
-
-    // Wrap player into game world
-    if (player.x > game.world.bounds.width) player.x = 0;
-    if (player.x < 0) player.x = game.world.bounds.width;
-    if (player.y > game.world.bounds.height) player.y = 0;
-    if (player.y < 0) player.y = game.world.bounds.height;
-        
-    game.debug.body(player);
-
+  // Move/turn toward pointer
+  game.physics.arcade.moveToPointer(player, 400);
+  player.rotation = game.physics.arcade.angleToPointer(player);
+
+  // Stop on mouseover
+  if (Phaser.Rectangle.contains(player.body, game.input.worldX, game.input.worldY)) {
+    player.body.velocity.setTo(0, 0);
+  }
+
+  if (trailCounter++ > trailInterval) {
+    trailCounter = 0;
+
+    // Draw trail
+    var trail = game.add.graphics(0, 0);
+    for (let x of [0 - game.world.bounds.width, 0, game.world.bounds.width])
+      for (let y of [0 - game.world.bounds.height, 0, game.world.bounds.height]) {
+        trail.beginFill(0xffffff);
+        trail.drawCircle(x + player.x, y + player.y, 3);
+        trail.endFill();
+      }
+    setTimeout(() => trail.kill(), 3000); // Keep trail 3 seconds long
+  }
+
+  // Wrap player into game world
+  if (player.x > game.world.bounds.width) player.x = 0;
+  if (player.x < 0) player.x = game.world.bounds.width;
+  if (player.y > game.world.bounds.height) player.y = 0;
+  if (player.y < 0) player.y = game.world.bounds.height;
+
+  game.debug.body(player);
+
+  pushStateToServer();
```

Comment:
I have an auto-formatter plugin for VSCode JS. That's what all these changes are from